### ISSUE TITLE
Inconsistency between the paper and the code

### ISSUE THREAD

**Issue by JPcooldev** [Davidup1/URLBERT#3]
In the paper, you state that the loss function in pre-training stage 2 is computed as $L=L_{MLM}+L_{SCL}+10*L_{VAT}$, but in the code you have:
```python
loss = adv_loss + 10 * loss
```
Which version did you actually use?
